packages/react-native/src/theme/createReactNativeTheme.ts

```diff
@@ -0,0 +1,55 @@
+// Internal Style Dictionary methods
+import deepExtend from 'style-dictionary/lib/utils/deepExtend';
+import get from 'lodash/get';
+import mapValues from 'lodash/mapValues';
+
+import { defaultTheme } from './defaultTheme';
+import { ColorMode, Theme, ReactNativeTheme } from './types';
+import { Tokens } from './tokens';
+
+const mapValuesDeep = (v: object | string, callback: Function): object =>
```

Comment:
Please use meaningfully named params for easier reading and maintenance in the future